### DISCUSSION TITLE
Handling callable fields in Default resolver

### DISCUSSION THREAD

**Discussion by heikkiket** [mirumee/ariadne#691] [Q&A] [answered]
Per [documentation](https://ariadnegraphql.org/docs/resolvers.html#default-resolver), this should work:
```
type_def = """
    type User {
        username: String!
        likes: Int!
        initials(length: Int!): String
    }
"""

class UserObj:
    username = "admin"

    def likes(self):
        return count_user_likes(self)

    def initials(self, length):
        return self.username[:length]
```

However, when I ran this example and tried making a query like this:
```
{
 user {
   likes
 }
}
```

I got an error:
`likes() takes 1 positional argument but 2 were given`

When I changed likes method to be:
`def likes(self, info)`
it seemed to work.

Is there a fault in the documentation, or did I miss something?

**Comment by rafalp** [comment, marked as answer]
Looks like a bug in Ariadne where default resolver is passing `info` to resolved function.

**Comment by heikkiket** [comment]
I made an issue about this.
https://github.com/mirumee/ariadne/issues/692

I suppose we can continue discussion there. I'm interested in helping to fix this.
